_maps/shuttles/syndicate/syndicate_litieguai.dmm

```diff
@@ -1424,551 +1665,440 @@
 	dir = 1
 	},
-/obj/structure/window/reinforced,
-/obj/machinery/door/window/northright,
-/obj/structure/cable{
-	icon_state = "0-2"
+/turf/open/floor/engine,
+/area/ship/storage)
+"JQ" = (
+/obj/structure/sign/warning/gasmask{
+	pixel_y = 32
 	},
-/obj/machinery/door/poddoor{
-	id = "traumaenginer"
+/obj/item/stack/marker_beacon/thirty,
+/obj/item/stack/marker_beacon/thirty,
+/obj/item/stack/marker_beacon/thirty,
+/obj/item/stack/marker_beacon/thirty,
+/obj/structure/rack,
+/obj/item/pickaxe/emergency{
+	desc = "For extracting yourself from rough landings, and getting to the even rougher ones";
+	name = "Medical Retrieval Tool"
 	},
-/turf/open/floor/plating,
-/area/ship/maintenance/starboard)
-"OA" = (
-/obj/machinery/firealarm/directional/west,
-/obj/machinery/airalarm/directional/south,
-/obj/effect/turf_decal/industrial/hatch/red,
-/obj/structure/closet/crate/freezer/surplus_limbs/organs,
+/obj/item/pickaxe/emergency{
+	desc = "For extracting yourself from rough landings, and getting to the even rougher ones";
+	name = "Medical Retrieval Tool"
+	},
+/obj/item/pickaxe/emergency{
+	desc = "For extracting yourself from rough landings, and getting to the even rougher ones";
+	name = "Medical Retrieval Tool"
+	},
+/obj/effect/turf_decal/industrial/outline/red,
+/obj/machinery/firealarm/directional/east,
+/turf/open/floor/plasteel/tech/grid,
+/area/ship/storage)
+"Kc" = (
+/obj/structure/sign/departments/medbay/alt,
+/turf/closed/wall/mineral/titanium/nodiagonal,
+/area/ship/crew)
+"Ki" = (
+/obj/effect/turf_decal/industrial/outline/red,
+/obj/machinery/vending/medical,
 /turf/open/floor/plasteel/tech,
 /area/ship/medical)
-"OB" = (
-/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
-	dir = 4
+"Kk" = (
+/obj/structure/window/reinforced{
+	dir = 1
 	},
-/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
+/obj/machinery/door/window/northright{
 	dir = 4
 	},
+/obj/machinery/power/terminal{
+	dir = 8
+	},
 /obj/structure/cable{
-	icon_state = "4-8"
+	icon_state = "0-4"
 	},
-/obj/machinery/light/small/directional/north,
+/obj/machinery/light/small/directional/south,
 /turf/open/floor/plating,
-/area/ship/maintenance/starboard)
-"OK" = (
-/turf/closed/wall/mineral/titanium/nodiagonal,
-/area/ship/storage)
-"Po" = (
-/obj/machinery/holopad/emergency/medical,
-/obj/effect/turf_decal/spline/fancy/opaque/black{
-	dir = 1
-	},
-/turf/open/floor/plasteel/tech,
-/area/ship/medical)
-"Pp" = (
-/obj/effect/turf_decal/corner/opaque/white/mono,
-/turf/open/floor/plasteel/white,
-/area/ship/medical)
-"Pr" = (
-/obj/effect/turf_decal/trimline/opaque/red/filled/warning{
-	dir = 8
-	},
-/obj/machinery/atmospherics/pipe/manifold4w/scrubbers/hidden/layer4,
-/obj/machinery/atmospherics/pipe/manifold/supply/hidden/layer2{
+/area/ship/maintenance/port)
+"Kp" = (
+/obj/machinery/atmospherics/components/unary/vent_scrubber/on/layer4{
 	dir = 4
 	},
-/obj/structure/cable{
-	icon_state = "2-8"
+/obj/machinery/airalarm/directional/east{
```

Comment:
This air alarm (I think; It's the one in the starboard nacelle anyway) is a rogue air alarm detached from the wall.